src/components/TimeseriesStatistics.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <table class="table">
+    <caption>
+      Statistiken
+    </caption>
+    <thead>
+      <th>Station</th>
+      <th>Parameter</th>
+      <th class="text-end">min</th>
+      <th class="text-end">mean</th>
+      <th class="text-end">max</th>
+      <th class="text-end">sum</th>
```

Comment:
```suggestion
      <th class="text-end">Min</th>
      <th class="text-end">⌀</th>
      <th class="text-end">Max</th>
      <th class="text-end">Summe</th>
```

Reply:
Thanks, that looks better and less hacky ;)